popup/folderlist.js

```diff
@@ -291,7 +296,7 @@ class TBFolderList extends HTMLElement {
     }
 
     this.selected = item;
-    this.shadowRoot.querySelector(".folder-list-body").focus();
+    //this.shadowRoot.querySelector(".folder-list-body").focus();
```

Comment:
Curious why you removed this? I think this was needed to make it possible to navigate with the keyboard.

Reply:
As you see I put this in another PR. But I guess when I prepared this PR I already this comment in. If you want I can edit this PR.

Reply:
Yes, would appreciate if you could remove any commented out code from this PR. We can try to find a solution in your other PR for the focus issue.